src/components/TodosList.tsx

```diff
@@ -0,0 +1,22 @@
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+import { TodoInfo } from './TodoInfo';
+
+type Props = {
+  todos: Todo[]
+};
+
+export const TodosList: React.FC<Props> = ({ todos }) => {
```

Comment:
Include the filter status prop in the destructuring assignment for the function component.